src/result.ts

```diff
@@ -447,6 +459,10 @@ export class Err<T, E> implements IResult<T, E> {
     return v
   }
 
+  unwrapOrElse<A>(err: (e: E) => A): T | A {
+    return err(this.error)
+  }
```

Comment:
I still find the naming `err` to be a bit confusing, but now I see that it's used everywhere for this type of argument 🤷🏼 e.g. in match directly below. no reason to break that pattern...